io/io/inc/ROOT/RRawFile.hxx

```diff
@@ -160,6 +160,8 @@ public:
    size_t Read(void *buffer, size_t nbytes);
    /// Change the cursor fFilePos
    void Seek(std::uint64_t offset);
+   /// Returns the offset for the next Read/Readln call
+   std::uint64_t GetFilePos() const { return fFilePos; }
```

Comment:
Exposing FilePos (and Seek) means than intrinsically RRawFile is thread unsafe.  This is likely 'fine' but we need to make sure that we document it (I.e. a thread safe file accessor would need to be build on to of it)

Reply:
I don't follow..why would users expect that the same RRawFile can be used from multiple threads concurrently? That's never an implied guarantee.

Reply:
We have a few request for TFile to be thread-safe.   Not being thread-safe means the client needs to do the lock explicitly (eg. TTree, RDataFrame, etc).  

Eitherway, it never hurts to be explicit :)

Reply:
Thanks, clarified in the RRawFile class comment.